modules/tensor_mechanics/src/materials/ADSingleVariableReturnMappingSolution.C

```diff
@@ -393,7 +395,7 @@ ADSingleVariableReturnMappingSolution<compute_stage>::outputIterationStep(
                  << " trial_stress=" << MetaPhysicL::raw_value(effective_trial_stress)
                  << " scalar=" << MetaPhysicL::raw_value(scalar) << " residual=" << residual
                  << " ref_res=" << reference_residual
-                 << " rel_res=" << std::abs(MetaPhysicL::raw_value(residual)) / reference_residual
+                 << " rel_res=" << std::abs(residual) / reference_residual
```

Comment:
why did you remove that raw_value?

Reply:
It's already converted a couple lines up